finagle/h2/src/test/scala/com/twitter/finagle/buoyant/h2/netty4/Netty4StreamTransportTest.scala

```diff
@@ -313,8 +311,8 @@ class Netty4StreamTransportTest extends FunSuite {
     assert(!transport.onReset.isDefined)
     assert(!transport.isClosed)
 
-    reqStream.reset(Reset.InternalError)
-    assert(await(endF.liftToTry) == Throw(StreamError.Local(Reset.InternalError)))
```

Comment:
We now use the StreamError type only in `onReset` to signal to the dispatcher if a reset needs to be written to the remote.